crates/sui-bridge-indexer/src/storage.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Error};
+use async_trait::async_trait;
+use diesel::dsl::now;
+use diesel::{ExpressionMethods, TextExpressionMethods};
+use diesel::{OptionalExtension, QueryDsl, SelectableHelper};
+use diesel_async::scoped_futures::ScopedFutureExt;
+use diesel_async::AsyncConnection;
+use diesel_async::RunQueryDsl;
+
+use crate::postgres_manager::PgPool;
+use crate::schema::progress_store::{columns, dsl};
+use crate::schema::{sui_error_transactions, token_transfer, token_transfer_data};
+use crate::{models, schema, ProcessedTxnData};
+use sui_indexer_builder::indexer_builder::{IndexerProgressStore, Persistent};
+use sui_indexer_builder::Task;
+
+/// Persistent layer impl
+#[derive(Clone)]
+pub struct PgBridgePersistent {
+    pool: PgPool,
+}
+
+impl PgBridgePersistent {
+    pub fn new(pool: PgPool) -> Self {
+        Self { pool }
+    }
+}
+
+// TODO: this is shared between SUI and ETH, move to different file.
```

Comment:
does this comment still applies?

Reply:
no, and i removed it in one of the following PRs